WooCommerce/Classes/ViewRelated/Orders/OrdersMasterViewController.swift

```diff
@@ -0,0 +1,218 @@
+
+import UIKit
+import struct Yosemite.OrderStatus
+import struct Yosemite.Note
```

Comment:
I'm also interested in this. There is any speed improvement in doing this?

Reply:
As far as I know, there is no build improvement when doing this. The compiler should be smart enough to ignore those which are not declared in the file. 

This specific importing is sort of a personal experiment. A few reasons are as follows.

#### Clarifying Intention 

I always keep this concept in mind in every code that I write. I want my code to be easily understood and unambiguous. Though, I admit, I fail quite a lot with this. 😄

In this case, I wanted future readers of `OrdersMasterViewController` to know that this "view" will only operate with `OrderStatus` and `Note` models. They shouldn't expect anything else. More importantly, they shouldn't expect `Order` to even be used in here. 

#### Reducing Scope

I wanted to keep this class as focused as possible — single responsibility principle. To protect myself, I can reduce the available symbols by using the individual `import`. 

![image](https://user-images.githubusercontent.com/198826/73963560-c17e5900-48cd-11ea-82a5-192cd4359508.png)

![image](https://user-images.githubusercontent.com/198826/73963600-db1fa080-48cd-11ea-86cd-7f96c8628100.png)

If I start using `Order` in this class, since I am forced to _write_ the `import`, I also force myself to think whether this actually makes sense. 

----

With all that said, I am not sure if this experiment will last. 😃 It takes a while to write these `import` statements. Those precious seconds can be used somewhere else, like thinking. 😄